.github/workflows/ci.yaml

```diff
@@ -27,11 +24,11 @@ jobs:
     needs: lint
     strategy:
       matrix:
-        node-version: [14, 16, 'lts/*']
+        node-version: [18, 20, 'lts/*']
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
     - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v3
+      uses: actions/setup-node@v4
```

Comment:
You could use Dependatbot to update actions. You would have to add another [package-ecosystem](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem) for `github-actions`.

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file


```yaml
./github/dependabot.yml
version: 2
updates:
  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "daily"
```